src/reducers/reducer.js

```diff
@@ -20,6 +20,7 @@ const reducer = combineReducers({
     dependenciesRequest: API.dependenciesRequest,
     layoutRequest: API.layoutRequest,
     loginRequest: API.loginRequest,
+    reloadHash: API.reloadRequest,
```

Comment:
Any reason why it's named `reloadHash` here and `reloadRequest` on `API.reloadRequest`?

Reply:
Good catch, I refactored `reloadHash` to `reloadRequest` to conform to the other API, I guess that one didn't follow up.